### DISCUSSION TITLE
March 7th(Honkai Star Rail) Voice not working

### DISCUSSION THREAD

**Discussion by nightfall-love** [pipeeeeees/waze-voicepack-links#23] [General]
Hi there! There seems to be an issue with March 7th's voice pack.

I followed all the intructions, added her voice but having her speeking normally while driving is not working. It defaults to English US - Amy. 

Is there a posibile solution to fix it?

**Comment by pipeeeeees** [comment]
This is a general issue with all custom voicepacks on the Waze application specifically on devices running android. You can try following the debugging steps in the readme of this repo, but some have shared that this does not work for them (it has for others). 

You can browse the r/waze subreddit for posts about this going back many months. One for example: https://www.reddit.com/r/waze/comments/14k8383/so_wazes_voices_are_kinda_messed_up_right_now_on/

You can try clearing cache by entering in `cc@tts` into the Waze in-app search bar (or maybe `##@cctts`... they changed it at some point), but if that does not work then I am unsure what can help other than downgrading android to 12 or below or getting an iOS device. Drop a reply here if you manage to find a fix - it may help others.
